apps/src/templates/levelSummary/FreeResponseResponses.jsx

```diff
@@ -21,8 +21,14 @@ const FreeResponseResponses = ({
   responses,
   showStudentNames,
   eventData,
+  scriptName,
```

Comment:
nit: general request to please prefer to call these "units", unless you are working in code that is already quite heavy on "scripts"

Reply:
thank you for the reminder!

Reply:
🤔 Is this preference all the way down the stack? Like are you suggesting `scriptId` in the api layer too?

Reply:
yes, the preference is all the way down the stack, however it is just "best effort". if it is less confusing to use "script" somewhere because that's whats already being used then it is ok to stick with it. script_id is a tough one because I don't think we renamed that particular field in active record.